app/src/test/java/org/mozilla/fenix/settings/logins/SavedLoginsStorageControllerTest.kt

```diff
@@ -32,16 +31,15 @@ import org.mozilla.fenix.settings.logins.fragment.EditLoginFragmentDirections
 
 @RunWith(FenixRobolectricTestRunner::class)
 class SavedLoginsStorageControllerTest {
-    val testDispatcher = TestCoroutineDispatcher()
     @get:Rule
-    val coroutinesTestRule = MainCoroutineRule(testDispatcher)
+    val coroutinesTestRule = MainCoroutineRule()
 
     private val passwordsStorage: SyncableLoginsStorage = mockk(relaxed = true)
     private lateinit var controller: SavedLoginsStorageController
     private val navController: NavController = mockk(relaxed = true)
     private val loginsFragmentStore: LoginsFragmentStore = mockk(relaxed = true)
-    private val scope = TestCoroutineScope()
-    private val ioDispatcher = TestCoroutineDispatcher()
+    private val ioDispatcher = coroutinesTestRule.testDispatcher
+    private val scope = TestCoroutineScope(ioDispatcher)
```

Comment:
maybe move these below line 35, so we can group the coroutines and not have it in the middle of these mockk() declarations.